ASTAnalyses/ASTInfo/Specifiers/SpecifierManager.py

```diff
@@ -0,0 +1,46 @@
+from ASTAnalyses.ASTInfo.Specifiers.Specifier import Specifier
+from ASTAnalyses.ASTInfo.Specifiers.FunctionStructure.Function import Function
+from ASTAnalyses.ASTInfo.Specifiers.StructStructure.Struct import Struct
+from ASTAnalyses.ASTInfo.Debug import *
+
+
+class SpecifierManager:
+    """Manages a list of specifiers
+    """
+
+    def __init__(self):
+        self.__specifiers: list[Specifier] = []
+
+    # if struct is not found, a new one is returned with name struct_name if error_if_not_found is false
```

Comment:
It used to be a parameter that existed but was removed because it was ultimately unused; I'll update comment to remove the mention of `error_if_not_found`